judge/highlight_code.py

```diff
@@ -7,13 +7,6 @@ def _make_pre_code(code):
     return mark_safe('<pre>' + escape(code) + '</pre>')
 
 
-def _wrap_code(inner):
-    yield 0, '<code>'
-    for tup in inner:
-        yield tup
-    yield 0, '</code>'
-
-
 try:
     import pygments
     import pygments.lexers
```

Comment:
(This comment is for line 13)
Change to `import pygments.formatters`. The `.html` is a historical artifact.

Reply:
Done